Sdk/Speckle.Connectors.Utils/Operations/ReceiveOperation.cs

```diff
@@ -29,26 +30,38 @@ IServerTransportFactory serverTransportFactory
     Action<string, double?>? onOperationProgressed = null
   )
   {
+    using var execute = SpeckleActivityFactory.Start();
+    Speckle.Core.Api.GraphQL.Models.Version? version;
+    Base? commitObject;
+    HostObjectBuilderResult? res;
     // 2 - Check account exist
     Account account = _accountService.GetAccountWithServerUrlFallback(receiveInfo.AccountId, receiveInfo.ServerUrl);
-
-    // 3 - Get commit object from server
     using Client apiClient = new(account);
-    var version = await apiClient
-      .Version.Get(receiveInfo.SelectedVersionId, receiveInfo.ModelId, receiveInfo.ProjectId, cancellationToken)
-      .ConfigureAwait(false);
+    using (var _ = SpeckleActivityFactory.Start("Receive from server"))
```

Comment:
a better message would be "Fetching Version Info"

Since this `Version` object is only lightweight info about the `Version`, not any object data.

Reply:
I also wonder if we'd prefer this to be implemented inside the client....
as a "executing graphql request" trace on ALL graphql queries/mutations